dataproxy/vercel-cli/api/index.js

```diff
@@ -0,0 +1,72 @@
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient({})
+
+export default async (req, res) => {
+  const data = await prisma.$transaction([
+    prisma.user.findFirst(),
+    prisma.user.findMany()
+  ])
+
+  const json = JSON.stringify({ data })
```

Comment:
Why simplified script with no creation and file checking here?

Reply:
Yes - I did not make the connection that this is all the same database for the dataproxy tests.
We should have an issue to expand that. Writing is kind of important :D

Reply:
https://github.com/prisma/ecosystem-tests/issues/2874